docs/source/en/api/pipelines/text_to_video_zero.md

```diff
@@ -95,7 +90,7 @@ for i in range(len(chunk_ids)):
 # Concatenate chunks and save
 result = np.concatenate(result)
 result = [(r * 255).astype("uint8") for r in result]
-imageio.mimsave("video.mp4", result, fps=4)
+export_to_video(result, "video.mp4")
```

Comment:
Also just getting a green video here

Reply:
With `video_length = 48` and `chunk_size = 16`, this video was created:

https://github.com/huggingface/diffusers/assets/46008593/5134fd71-7df4-4c19-8e43-a4904c467370